ui/containers/WorkspaceRequest/components/OverviewPage.tsx

```diff
@@ -13,7 +13,7 @@ export default class OverviewPage extends React.Component<Props> {
     const { request, setRequest } = this.props;
     setRequest({
       ...request,
-      [key]: e.target.value,
+      [key]: e.target.value.trim().length ? e.target.value : '',
```

Comment:
what's the behavior here Shafi? Is an error thrown? The user should be required to fill in these fields before continuing.

Reply:
It won't allow user until type at least one character. Previously it was allowing space